EIPS/eip-173.md

```diff
@@ -0,0 +1,109 @@
+---
+eip: 173
+title: ERC-173 Contract Ownership Standard
+author: Nick Mudge <nick@perfectabstractions.com>, Dan Finlay <dan@danfinlay.com>
+type: Standards Track
+category: ERC
+status: Draft
+review-period-end: 2018-06-18
```

Comment:
Please omit this for drafts.

Reply:
Understood. I removed the "review-period-end".